core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -552,7 +552,7 @@ merge_parent_call_restrictions(Account) ->
                       {'error', _} -> Acc
                   end
           end,
-    lists:foldl(Fun, kz_json:new(), [kzd_accounts:id(Account) | kzd_accounts:tree(Account)]).
+    lists:foldl(Fun, Account, [kzd_accounts:id(Account) | kzd_accounts:tree(Account)]).
```

Comment:
no need to merge Account with Account.
i this should be `lists:foldl(Fun, Account, kzd_accounts:tree(Account)).`

Reply:
Ahh yes! This makes sense!